src/components/CopyToClipboard.tsx

```diff
@@ -4,10 +4,15 @@ import { Box } from "@chakra-ui/react"
 
 export interface IProps {
   text: string
+  inline?: boolean
   children: (isCopied: boolean) => React.ReactNode
 }
 
-const CopyToClipboard: React.FC<IProps> = ({ children, text }) => {
+const CopyToClipboard: React.FC<IProps> = ({
+  children,
+  text,
+  inline = false,
```

Comment:
Lets try to minimize the amount of props to keep the component as simple as possible. What do you think of the following idea:

```tsx
// 1. extend the props
interface IProps extends ChakraProps {
  ...
}

// 2. spread the rest of the props in the parent element
<Box ref={targetEl} cursor="pointer" {...props}>
```

Now from the consumer side you could use chakra props to do what you want to do
```tsx
<CopyToClipboard display="inline">
```

Reply:
Sure... didn't want to creep into other components too much but I can fix this one up